typing/mode_intf.mli

```diff
@@ -541,6 +541,10 @@ module type S = sig
             output list exactly once. *)
         val to_list : t -> atom list
 
+        (** Builds up a modality from a list of [atom], by composing each atom with
+            identity. *)
```

Comment:
```suggestion
            identity. The modalities are applied left to right. *)
```

Check me on this, but I think that's what the `fold_left` will do. Obviously different axes commute, but there's nothing stopping us from having multiple `atom`s on one axis (which won't necessarily commute).